### PR TITLE
[zh-cn]: init the translation of "CSS box alignment"

### LINKED ISSUE #16857: Translate css/css_box_alignment/box_alignment_in_block_abspos_tables
### MDN URL

https://github.com/mdn/content/blob/main/files/en-us/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md

### 与此议题相关的小节或标题？

Translate it.

### 哪些信息是错误、无用或不完善的？

no

### 你期望看到什么？

Chinese content.

### 你有什么支持的链接、参考或引用？

css/css_box_alignment/box_alignment_in_block_abspos_tables

### 你还想补充什么内容？

link: https://github.com/mdn/translated-content/pull/16858

### MDN metadata

_No response_

### PR DESCRIPTION
**Pull request by zhanghengxin**
…css_box_alignment/box_alignment_in_block_abspos_tables/index.md

fix: https://github.com/mdn/translated-content/issues/16857

### CODE REVIEW

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- {{cssxref("justify-content")}}
+- {{cssxref("align-content")}}
+- {{cssxref("justify-self")}}
+- {{cssxref("align-self")}}
+
+### 词汇条目
+
+- [Alignment subject](https://developer.mozilla.org/en-US/docs/Glossary/Alignment_Subject)
+- [Alignment container](https://developer.mozilla.org/en-US/docs/Glossary/Alignment_Subject)
+- [Fallback alignment](https://developer.mozilla.org/en-US/docs/Glossary/Fallback_Alignment)
```
[the remark this answers is not in the document]
Comment (zhanghengxin) [reply files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:63]: /zh-CN/docs/Glossary/Fallback_Alignment ,  They dont exist.

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+title: 块状、绝对定位和表格布局中的盒子对齐
```
Comment (yin1999) [inline files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:2]: Please check: #12632

```suggestion
title: 区块、绝对定位和表格布局中的盒子对齐
```

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+slug: Web/CSS/CSS_box_alignment/Box_alignment_in_block_abspos_tables
+---
+
+[Box对齐规格书](/zh-CN/docs/Web/CSS/CSS_box_alignment) 详细介绍了对齐在各种布局方法中的工作原理。在本页,我们研究 box 对齐在块布局上下文中的工作方式,包括浮动、定位和表格元素。由于本页旨在详细介绍专门与块布局和 box 对齐相关的内容,它应与主要的 [Box 对齐](/zh-CN/docs/Web/CSS/CSS_box_alignment) 页面一起阅读,该页面详细介绍了 box 对齐在各种布局方法中的共同特点。
```
Comment (yin1999) [inline files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:6]:
```suggestion
[盒子对齐规范](/zh-CN/docs/Web/CSS/CSS_box_alignment)详细说明了对齐在各种布局方法中的工作原理。在本页，我们研究盒子对齐在块布局上下文中的工作方式，包括浮动、定位和表格元素。由于本页旨在详细介绍专门与块布局和盒子对齐相关的内容，它应与主要的[盒子对齐](/zh-CN/docs/Web/CSS/CSS_box_alignment)页面一起阅读，该页面详细介绍了盒子对齐在各种布局方法中的共同特性。
```

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+> **注意:**在撰写本文时(2018 年 5 月),块布局中实际上还不支持 box 对齐属性。本文详细介绍了这些属性在规范中如何期望被实现的情况,以求完整性,并可能会随着规范和浏览器实现的发展而改变。
```
Comment (yin1999) [inline files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:8]:
```suggestion
> **备注：** 在撰写本文时（2018 年 5 月），块布局中实际上还不支持盒子对齐属性。本文详细介绍了这些属性在规范中如何期望被实现的情况，以求完整性，并可能会随着规范和浏览器实现的发展而改变。
```

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+## align-content 和 justify-content
+
+{{cssxref("justify-content")}} 属性不适用于块容器或表格单元格。
+
+{{cssxref("align-content")}} 属性适用于块轴,以对容器中的内容进行对齐。如果请求了一个内容分布方法如 `space-between`、`space-around` 或 `space-evenly`,那么将使用后备对齐,因为内容被视为一个[对齐主体](/zh-CN/docs/Glossary/Alignment_Subject)。
```
Comment (yin1999) [inline files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:14]:
```suggestion
{{cssxref("align-content")}} 属性适用于块向轴，以对容器中的内容进行对齐。如果请求了一个内容分布方法如 `space-between`、`space-around` 或 `space-evenly`，那么将使用后备对齐，因为内容被视为一个[对齐主体](/zh-CN/docs/Glossary/Alignment_Subject)。
```

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+## justify-self
+
+{{cssxref("justify-self")}} 属性用于在内联轴上对包含块中的项目进行对齐。
```
Comment (yin1999) [inline files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:18]:
```suggestion
{{cssxref("justify-self")}} 属性用于在行向轴上对包含块中的项目进行对齐。
```

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+此属性不适用于浮动元素或表格单元格。
+
+## align-self
+
+{{cssxref("align-self")}} 属性不适用于块级框(包括浮动),因为块轴上有多个项目。它也不适用于表格单元格。
```
Comment (yin1999) [inline files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:24]:
```suggestion
{{cssxref("align-self")}} 属性不适用于块级框（包括浮动），因为块轴上有多个项目。它也不适用于表格单元格。
```

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+### 使用绝对定位的元素
+
+对齐容器是带有 top、left、bottom 和 right 偏移值的定位块。正常关键字解析为 `stretch`,除非定位项目是一个替换元素,在这种情况下它解析为 `start`。
```
Comment (yin1999) [inline files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:28]:
```suggestion
对齐容器是带有 top、left、bottom 和 right 偏移值的定位块。正常关键字解析为 `stretch`，除非定位项目是一个替换元素，在这种情况下它解析为 `start`。
```

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+## 在这些布局方法中进行对齐的现状
+
+由于我们当前没有浏览器支持块布局中的 box 对齐,你对齐的选择是使用现有的对齐方法,或者使容器内的单个项目成为弹性项目,以利用flexbox中的对齐属性如规范中指定的那样。
+
+块水平对齐在 flexbox 之前通常通过对块设置自动边距来实现。{{cssxref("margin")}}值为`auto` 会吸收该维度中所有的可用空间,因此通过为块设置左右自动边距可以将其 centered:
```
Comment (yin1999) [inline files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:34]:
```suggestion
由于我们当前没有浏览器支持块布局中的 box 对齐,你对齐的选择是使用现有的对齐方法，或者使容器内的单个项目成为弹性项目，以利用弹性盒子中的对齐属性如规范中指定的那样。

区块水平对齐在弹性盒子之前通常通过对块设置自动边距来实现。{{cssxref("margin")}} 值为`auto` 会吸收该维度中所有的可用空间，因此通过为块设置左右自动边距可以将其居中：
```

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+
+```css
+.container {
+  width: 20em;
+  margin-left: auto;
+  margin-right: auto;
+}
+```
+
+在表格布局中,你可以使用{{cssxref("vertical-align")}}属性对单元格中的内容在该单元格内进行对齐。
```
Comment (yin1999) [inline files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:44]:
```suggestion
在表格布局中，你可以使用 {{cssxref("vertical-align")}} 属性对单元格中的内容在该单元格内进行对齐。
```

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+
+对于许多使用场景,将块容器转换为弹性项目将给你 looking for 的对齐能力。在下面的示例中,含有单个项目的容器已转换为弹性容器,以便能够使用对齐属性。
```
Comment (yin1999) [inline files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:46]:
```suggestion
对于许多使用场景，将块容器转换为弹性项目将为你提供所需的对齐能力。在下面的示例中，含有单个项目的容器已转换为弹性容器，以便能够使用对齐属性。
```

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+
+{{EmbedGHLiveSample("css-examples/flexbox/alignment/intro.html", '100%', 700)}}
+
+## 引用
```
Comment (yin1999) [inline files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:50]:
```suggestion
## 参考
```

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+
+### CSS 属性
+
+- {{cssxref("justify-content")}}
+- {{cssxref("align-content")}}
+- {{cssxref("justify-self")}}
+- {{cssxref("align-self")}}
+
+### 词汇条目
```
Comment (yin1999) [inline files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:59]:
```suggestion
### 术语条目
```

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md (hunk 1 above)
Comment (yin1999) [inline files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:63]:
```suggestion
- [对齐主体](/zh-CN/docs/Glossary/Alignment_Subject)
- [对齐容器](/zh-CN/docs/Glossary/Alignment_Subject)
- [后备对齐](/zh-CN/docs/Glossary/Fallback_Alignment)
```

**Review by yin1999 — COMMENTED** [review]
Thanks for the contribution. But we still need to fix some errors

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md (hunk 1 above)
Comment (zhanghengxin) [reply files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:63]: mark: /zh-CN/docs/Glossary/Alignment_Subject, These paths do not exist.

File: files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+
+### 使用绝对定位的元素
+
+对齐容器是带有 top、left、bottom 和 right 偏移值的定位块。正常关键字解析为 `stretch`，除非定位项目是一个替换元素，在这种情况下它解析为 `start`。
+
+## 在这些布局方法中进行对齐的现状
+
+由于我们当前没有浏览器支持块布局中的 box 对齐,你对齐的选择是使用现有的对齐方法，或者使容器内的单个项目成为弹性项目，以利用弹性盒子中的对齐属性如规范中指定的那样。
+
+区块水平对齐在弹性盒子之前通常通过对块设置自动边距来实现。{{cssxref("margin")}} 值为`auto` 会吸收该维度中所有的可用空间，因此通过为块设置左右自动边距可以将其居中：
```
Comment (yin1999) [inline files/zh-cn/web/css/css_box_alignment/box_alignment_in_block_abspos_tables/index.md:34]:
```suggestion
由于我们当前没有浏览器支持块布局中的盒子对齐，因此你可以选择使用现有的对齐方法之一，或者将容器内的单个项目设置为弹性项目，以使用弹性盒子中指定的的对齐属性。

区块水平对齐在弹性盒子之前通常通过对块设置自动边距来实现。{{cssxref("margin")}} 值为 `auto` 会吸收该维度中所有的可用空间，因此通过为块设置左右自动边距可以将其居中：
```

**Review by yin1999 — APPROVED** [review]
Thanks @zhanghengxin. Welcome aboard @zhanghengxin 🎉

**Pull request merged**